distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java

```diff
@@ -397,6 +397,10 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
 
     protected Option[] configureAdditionalBundles() {
         return options(junitBundles(),
+                wrappedBundle(mavenBundle("ddf.catalog.registry",
```

Comment:
Refactored itests so that we don't need to directly interact with the catalog framework. Left some placeholder tests in there for when we have access to the federation admin service to do destination testing.